### YamlMime:UniversalReference
ms.openlocfilehash: 6cb0f3f80bc511a0865f3a5609cf1784ff077b9b
ms.sourcegitcommit: 87f95d58ec8de16e115bc344efeb084afc346b74
ms.translationtype: MT
ms.contentlocale: zh-TW
ms.lasthandoff: 08/12/2018
ms.locfileid: "40052187"
items:
- uid: azure-event-hubs.ErrorNameConditionMapper
  name: ErrorNameConditionMapper
  fullName: ErrorNameConditionMapper
  children:
  - azure-event-hubs.ErrorNameConditionMapper.ArgumentError
  - azure-event-hubs.ErrorNameConditionMapper.UnauthorizedError
  - azure-event-hubs.ErrorNameConditionMapper.ConnectionForcedError
  - azure-event-hubs.ErrorNameConditionMapper.ConnectionRedirectError
  - azure-event-hubs.ErrorNameConditionMapper.DecodeError
  - azure-event-hubs.ErrorNameConditionMapper.DetachForcedError
  - azure-event-hubs.ErrorNameConditionMapper.ErrantLinkError
  - azure-event-hubs.ErrorNameConditionMapper.FrameSizeTooSmallError
  - azure-event-hubs.ErrorNameConditionMapper.FramingError
  - azure-event-hubs.ErrorNameConditionMapper.HanldeInUseError
  - azure-event-hubs.ErrorNameConditionMapper.IllegalStateError
  - azure-event-hubs.ErrorNameConditionMapper.InternalServerError
  - azure-event-hubs.ErrorNameConditionMapper.InvalidFieldError
  - azure-event-hubs.ErrorNameConditionMapper.InvalidOperationError
  - azure-event-hubs.ErrorNameConditionMapper.LinkRedirectError
  - azure-event-hubs.ErrorNameConditionMapper.MessageLockLostError
  - azure-event-hubs.ErrorNameConditionMapper.MessageTooLargeError
  - azure-event-hubs.ErrorNameConditionMapper.ArgumentOutOfRangeError
  - azure-event-hubs.ErrorNameConditionMapper.NotImplementedError
  - azure-event-hubs.ErrorNameConditionMapper.OperationCancelledError
  - azure-event-hubs.ErrorNameConditionMapper.PreconditionFailedError
  - azure-event-hubs.ErrorNameConditionMapper.QuotaExceededError
  - azure-event-hubs.ErrorNameConditionMapper.ReceiverDisconnectedError
  - azure-event-hubs.ErrorNameConditionMapper.ResourceDeletedError
  - azure-event-hubs.ErrorNameConditionMapper.ResourceLockedError
  - azure-event-hubs.ErrorNameConditionMapper.SenderBusyError
  - azure-event-hubs.ErrorNameConditionMapper.ServerBusyError
  - azure-event-hubs.ErrorNameConditionMapper.ServiceCommunicationError
  - azure-event-hubs.ErrorNameConditionMapper.ServiceUnavailableError
  - azure-event-hubs.ErrorNameConditionMapper.SessionCannotBeLockedError
  - azure-event-hubs.ErrorNameConditionMapper.SessionLockLostError
  - azure-event-hubs.ErrorNameConditionMapper.SessionWindowViolationError
  - azure-event-hubs.ErrorNameConditionMapper.TransferLimitExceededError
  - azure-event-hubs.ErrorNameConditionMapper.UnattachedHandleError
  - azure-event-hubs.ErrorNameConditionMapper.MessagingEntityDisabledError
  langs:
  - typeScript
  type: enum
  summary: <span data-ttu-id="7e791-101">錯誤名稱來對應條件。</span><span class="sxs-lookup"><span data-stu-id="7e791-101">Maps the conditions to the Error names.</span></span>
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ArgumentError
  name: ArgumentError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-102">當收到不正確的引數時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-102">Error is thrown when an incorrect argument was received.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.UnauthorizedError
  name: UnauthorizedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-103">當連線參數錯誤，而且伺服器拒絕連線，則會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-103">Error is thrown when the connection parameters are wrong and the server refused the connection.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ConnectionForcedError
  name: ConnectionForcedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-104">操作員 intervened 因故關閉連線時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-104">Error is thrown when an operator intervened to close the connection for some reason.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ConnectionRedirectError
  name: ConnectionRedirectError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-105">當容器已無法再使用目前的連接上時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-105">Error is thrown when the container is no longer available on the current connection.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.DecodeError
  name: DecodeError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-106">無法解碼的資料時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-106">Error is thrown when data could not be decoded.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.DetachForcedError
  name: DetachForcedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-107">操作員 intervened 因故中斷連結時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-107">Error is thrown when an operator intervened to detach for some reason.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ErrantLinkError
  name: ErrantLinkError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-108">輸入收到的連結，並產生錯誤中卸離時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-108">Error is thrown when input was received for a link that was detached with an error.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.FrameSizeTooSmallError
  name: FrameSizeTooSmallError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-109">對等電腦無法傳送框架，因為會太大，協議的框架大小上限的框架內的最小的編碼方式，就的展演目前有效的值時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-109">Error is thrown when the peer cannot send a frame because the smallest encoding of the performative with the currently valid values would be too large to fit within a frame of the agreed maximum frame size.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.FramingError
  name: FramingError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-110">有效範圍標頭不能形成的傳入位元組資料流時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-110">Error is thrown when a valid frame header cannot be formed from the incoming byte stream.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.HanldeInUseError
  name: HanldeInUseError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-111">附加已收到使用已在使用附加連結的控制代碼時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-111">Error is thrown when an attach was received using a handle that is already in use for an attached link.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.IllegalStateError
  name: IllegalStateError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-112">對等體傳送的目前狀態中不允許的畫面格時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-112">Error is thrown when the peer sent a frame that is not permitted in the current state.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.InternalServerError
  name: InternalServerError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-113">發生內部伺服器錯誤時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-113">Error is thrown when an internal server error occured.</span></span> <span data-ttu-id="7e791-114">您可能會發現的 bug？</span><span class="sxs-lookup"><span data-stu-id="7e791-114">You may have found a bug?</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.InvalidFieldError
  name: InvalidFieldError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-115">在框架主體中，傳遞無效的欄位和作業無法繼續時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-115">Error is thrown when an invalid field was passed in a frame body, and the operation could not proceed.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.InvalidOperationError
  name: InvalidOperationError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-116">當作業嘗試，但不是允許，則會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-116">Error is thrown when an operation is attempted but is not allowed.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.LinkRedirectError
  name: LinkRedirectError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-117">無法解析以取得在目前的容器提供的地址時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-117">Error is thrown when the address provided cannot be resolved to a terminus at the current container.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.MessageLockLostError
  name: MessageLockLostError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-118">當訊息鎖定遺失時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-118">Error is thrown when the lock on the message is lost.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.MessageTooLargeError
  name: MessageTooLargeError
  children: []
  langs:
  - typeScript
  summary: '<span data-ttu-id="7e791-119">太大而無法傳送的訊息時，會擲回錯誤： 最大大小為 256 Kb。</span><span class="sxs-lookup"><span data-stu-id="7e791-119">Error is thrown when the message sent is too large: the maximum size is 256Kb.</span></span>'
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ArgumentOutOfRangeError
  name: ArgumentOutOfRangeError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-120">引數的值超出可接受的範圍，這個值時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-120">Error is thrown when an argument has a value that is out of the admissible range.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.NotImplementedError
  name: NotImplementedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-121">尚未實作功能，但預留位置存在時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-121">Error is thrown when a feature is not implemented yet but the placeholder is present.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.OperationCancelledError
  name: OperationCancelledError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-122">伺服器取消該作業，因為發生內部問題時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-122">Error is thrown when server cancels the operation due to an internal issue.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.PreconditionFailedError
  name: PreconditionFailedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-123">當應符合才能執行作業的條件不是，則會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-123">Error is thrown when a condition that should have been met in order to execute an operation was not.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.QuotaExceededError
  name: QuotaExceededError
  children: []
  langs:
  - typeScript
  summary: >-
    <span data-ttu-id="7e791-124">會擲回錯誤已超過 Azure 事件中樞服務匯流排配額。</span><span class="sxs-lookup"><span data-stu-id="7e791-124">Error is thrown the the Azure EventHub/ServiceBus quota has been exceeded.</span></span>

    <span data-ttu-id="7e791-125">會定期重設配額，這項作業會在那之前等候。</span><span class="sxs-lookup"><span data-stu-id="7e791-125">Quotas are reset periodically, this operation will have to wait until then.</span></span>

    <span data-ttu-id="7e791-126">傳訊實體已達到允許的大小上限。</span><span class="sxs-lookup"><span data-stu-id="7e791-126">The messaging entity has reached its maximum allowable size.</span></span>

    <span data-ttu-id="7e791-127">如果在個別取用者群組層級開啟的接收者數目已經達到上限 (5)，便可能發生此情況。</span><span class="sxs-lookup"><span data-stu-id="7e791-127">This can happen if the maximum number of receivers (which is 5) has already been opened on a per-consumer group level.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ReceiverDisconnectedError
  name: ReceiverDisconnectedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-128">當兩個或多個執行個體連接到相同的分割區使用不同的 epoch 值時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-128">Error is thrown when two or more instances connect to the same partition with different epoch values.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ResourceDeletedError
  name: ResourceDeletedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-129">已刪除用戶端正在與伺服器的實體時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-129">Error is thrown when a server entity the client is working with has been deleted.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ResourceLockedError
  name: ResourceLockedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-130">當用戶端嘗試使用伺服器的實體，並具有沒有存取權因為另一個用戶端可使用時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-130">Error is thrown when the client attempted to work with a server entity to which it has no access because another client is working with it.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.SenderBusyError
  name: SenderBusyError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-131">當用戶端傳送者沒有足夠的連結信用額度，傳送訊息時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-131">Error is thrown when the client sender does not have enough link credits to send the message.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ServerBusyError
  name: ServerBusyError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-132">伺服器太忙碌時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-132">Error is thrown when the server is busy.</span></span> <span data-ttu-id="7e791-133">呼叫端應稍待片刻，然後重試作業。</span><span class="sxs-lookup"><span data-stu-id="7e791-133">Callers should wait a while and retry the operation.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ServiceCommunicationError
  name: ServiceCommunicationError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-134">訊號相關訊息作業的一般通訊錯誤的錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-134">Error for signaling general communication errors related to messaging operations.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.ServiceUnavailableError
  name: ServiceUnavailableError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-135">當服務無法使用時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-135">Error is thrown when the service is unavailable.</span></span> <span data-ttu-id="7e791-136">應該重試作業。</span><span class="sxs-lookup"><span data-stu-id="7e791-136">The operation should be retried.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.SessionCannotBeLockedError
  name: SessionCannotBeLockedError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-137">無法鎖定 Azure 服務匯流排工作階段時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-137">Error is thrown when the Azure ServiceBus session cannot be locked.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.SessionLockLostError
  name: SessionLockLostError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-138">當 Azure 服務匯流排工作階段上的鎖定遺失時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-138">Error is thrown when the lock on the Azure ServiceBus session is lost.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.SessionWindowViolationError
  name: SessionWindowViolationError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-139">對等電腦違反工作階段的內送的視窗時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-139">Error is thrown when the peer violated incoming window for the session.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.TransferLimitExceededError
  name: TransferLimitExceededError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-140">對等體傳送更多的訊息傳輸超過目前允許的連結時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-140">Error is thrown when the peer sent more message transfers than currently allowed on the link.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.UnattachedHandleError
  name: UnattachedHandleError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-141">（其他非連結） 的畫面格收到參考目前不使用連接的連結的控制代碼時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-141">Error is thrown when a frame (other than attach) was received referencing a handle which is not currently in use of an attached link.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
- uid: azure-event-hubs.ErrorNameConditionMapper.MessagingEntityDisabledError
  name: MessagingEntityDisabledError
  children: []
  langs:
  - typeScript
  summary: <span data-ttu-id="7e791-142">嘗試存取/連線至已停用的傳訊實體時，會擲回錯誤。</span><span class="sxs-lookup"><span data-stu-id="7e791-142">Error is thrown when trying to access/connect to a disabled messaging entity.</span></span>
  type: field
  numericValue: 
  package: azure-event-hubs
